### DISCUSSION TITLE
Reading too much data in?

### DISCUSSION THREAD

**Discussion by Carlos-Fernands** [MilesCranmer/PySR#165] [Q&A]
After advice from Miles, I kept my training data way smaller than my original one:
- I had 800K rows of 12 features and reduced that to 2000 rows with 8 features.

All good here, except that I never got any model out, despite increasing the maxdepth, iterations, and other parameters.

Then I tried to increase the 2000 rows to 3000 (and later to 2500, with the same result) and that's where something "interesting" happened: I started to get this long traceback error message ending with:
".....exception: access violation reading 0x00....."

And I would have to restart the kernel, sometimes the whole Anaconda environment, because nothing else would work. 

Is there a genuine limitation about that amount of data being used? (> 2000 x 8 items)

**Comment by MilesCranmer** [comment]
Hey @Carlos-Fernands,
Sorry for the late reply, I didn't see this for some reason. 2000 rows is fine, I regularly use 5000 rows even on my laptop. It sounds like a bug - try updating Julia and PySR and run it again?
Cheers,
Miles
